### ISSUE TITLE
got `No nodes replied within time constraint` after command `celery -b redis://127.0.0.1:6379/0 status`

### ISSUE THREAD

**Issue by stormeyes** [celery/celery#5743]
Hi, my code as following, which is very simple:

```
import time
from celery import Celery

broker = 'redis://127.0.0.1:6379'
backend = 'redis://127.0.0.1:6379/0'

app = Celery('my_task', broker=broker, backend=backend)

@app.task
def add(x, y):
    time.sleep(5)
    return x + y
```

then I type command:
```
celery worker -A tasks --loglevel=info
```
and everything looks well, but when I type following command:
```
celery -A tasks inspect active
```
I got `Error: No nodes replied within time constraint.` and I type command:
```
celery -b redis://127.0.0.1:6379/0 status
```
and got the same error. and I type `celery inspect ping`, the command hang.....

this is what `celery report` output, and I am sure my disk space is quite large enough:
```
software -> celery:4.3.0 (rhubarb) kombu:4.6.4 py:3.7.0
            billiard:3.6.1.0 redis:3.3.8
platform -> system:Darwin arch:64bit
            kernel version:18.7.0 imp:CPython
loader   -> celery.loaders.default.Loader
settings -> transport:redis results:disabled
```

and this is my celery worker output
![image](https://user-images.githubusercontent.com/5072174/65386889-7840c900-dd73-11e9-8f57-30cf81a4c0a4.png)

So how can I fix or debug this problem?

**Comment by auvipy** [comment]
can you try with kombu==4.6.3?

**Comment by stormeyes** [comment]
> can you try with kombu==4.6.3?

OMG, after downgrade to 4.6.3, everything runs perfect......, thank u.

**Comment by auvipy** [comment]
this is actually a regression. this should be fixed in 4.6.5 release kombu, hopefully
